package.json

```diff
@@ -18,7 +18,9 @@
     "tape": "~2.3.2",
     "typedarray": "0.0.6"
   },
-  "bin": "./bin.js",
+  "bin": {
+    "shajs": "./bin.js"
```

Comment:
You could wait for https://github.com/yarnpkg/yarn/pull/7811 to be released. This might allow to fix this without a breaking change.

Reply:
Thanks man, this was already merged. I hope the author will finally merge this one here ?